litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/index.tsx

```diff
@@ -344,7 +344,6 @@ const AddProbe: React.FC<AddProbeProps> = ({
                 width="50%"
                 id="retry"
                 name="retry"
-                required
```

Comment:
@ispeakc0de  Can you please remove this line as well https://github.com/litmuschaos/litmus/blob/647b438466f30b07e887645ec97bc939b2d17dca/litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/index.tsx#L340